docs/extend/extend-apiml/api-mediation-security.md

```diff
@@ -180,25 +177,43 @@ The `auth/login` endpoint authenticates mainframe user credentials and returns a
 
 When authentication is successful, the response to the request is an empty body and a token is contained in a secure `HttpOnly` cookie named `apimlAuthenticationToken`. When authentication fails, the user receives a 401 status code.
 
+##### auth/query
+
+The full path of the `auth/query` endpoint appear as `https://{gatewayUrl}:{gatewayPort}/gateway/api/v1/auth/query` (preferred option) or `https://{gatewayUrl}:{gatewayPort}/api/v1/gateway/auth/query`.
+
 The `auth/query` endpoint validates the token and retrieves the information associated with the token.
 The query request requires the token through one of the following methods:
   * A cookie named `apimlAuthenticationToken`
   * Bearer authentication
 
 When authentication is successful, the response to the request is a JSON object which contains information associated with the token. When authentication fails, the user receives a 401 status code.
 
+##### auth/ticket
+
 The `auth/ticket` endpoint generates a PassTicket for the user associated with a token. The full path of the `auth/ticket` endpoint appears as https://{gatewayUrl}:{gatewayPort}/gateway/api/v1/auth/ticket` (preferred option) or `https://{gatewayUrl}:{gatewayPort}/api/v1/gateway/auth/ticket`.`
 
 This endpoint is protected by a client certificate.
 The ticket request requires the token in one of the following formats:
 
-- Cookie named apimlAuthenticationToken.
+- Cookie named `apimlAuthenticationToken`.
 - Bearer authentication
 
 The request takes the `applicationName` parameter, which is the name of the application for which the PassTicket should be generated. This parameter must be supplied.
 
 The response is a JSON object, which contains information associated with the ticket.
 
+##### auth/refresh
+
+**Note:** This endpoint is disabled by default. [Jwt token refresh endpoint enablement](../../user-guide/api-mediation/api-gateway-configuration.md#jwt-token-refresh-endpoint-enablement)
+
+The `auth/refresh` endpoint generates a new token for the user based on valid jwt token. The full path of the `auth/refresh` endpoint appears as `https://{gatewayUrl}:{gatewayPort}/gateway/api/v1/auth/refresh` (preferred option) or `https://{gatewayUrl}:{gatewayPort}/api/v1/gateway/auth/refresh`. The new token overwrites the old cookie with a `Set-Cookie` header. As part of the process, the old token gets invalidated and is not usable anymore.
```

Comment:
Do we need to mention that the old token gets invalidated? If it's something that is obvious, I would remove this line.

Reply:
It's not obvious, I think it should be mentioned explicitly.

Reply:
Because the token can be held by a third party, and in itself, it is still valid. It is only in the moment you want to use it that you hit a problem, as Gateway will recognize it as invalidated and reject it.